scania-digital-design-system/sdds
# feat(header): migrate header component

**Describe pull-request**  
Migration of header component.

**Solving issue**  
Fixes: [AB#2393](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2393)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> Header
3. Check that the design is aligned with the one in Figma. 

**Screenshots**  
-

**------------------NOTE------------------**
This was quite a large job with the theme variables, so please help me test it properly.
